### ISSUE TITLE
Need clarification

### ISSUE THREAD

**Issue by winternet-studio** [cyberzilla/DDEScanner#2]
Which scanners will this work with?

Is it correct understood that this is a program that will run in Windows and communicate with the scanner, get the scanned image and then upload it to a given URL in order to get the image into the web application?

How much does it cost?

**Comment by cyberzilla** [comment]
correct, but this version only suppot single result image (*.jpg) (not support duplex and multiple page right now), you can test on your scanner device while supporting wia (windows image acquisition), the demo and the price on app

cheers

**Comment by cyberzilla** [comment]
Stay tune, Soon I Will Update for small fix

**Comment by cyberzilla** [comment]
The Update Version is Released, enjoy

**Comment by winternet-studio** [comment]
Still no mention of the price... why do I need to run the program and go there to find it....

**Comment by cyberzilla** [comment]
Personal Use for 1 client PC is $355
Business Use for Unlimited PC is $3500

**Comment by cyberzilla** [comment]
The new version using twain driver, support ADF Scanner, Multiple Page with Duplex (2 side), the scanner result is PDF file.
